bears/c_languages/codeclone_detection/ClangCountingConditions.py

```diff
@@ -200,7 +200,7 @@ def _get_binop_operator(cursor):
                                  token.extent.end.column)):
             return token
 
-    return None  # pragma: no cover
+    return None  
```

Comment:
I haven't changed the codestyle. Tjis is the way it was writter. I have just removed the pragma no covers which were not required according to issue #1711